src/signals/settings/roles/components/RolesList/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@datapunt/asc-core';
+
+import LoadingIndicator from 'shared/components/LoadingIndicator';
+import ListComponent from 'components/List';
+
+import PageHeader from 'signals/settings/components/PageHeader';
+import formatRoles from '../../services/formatRoles';
+
+const StyledListComponent = styled(ListComponent)`
+  tr:nth-child(1),
```

Comment:
Is this correct? Both first `<tr>` and first `<td>` should have width `20%`?

Reply:
het was een typo:
  th:nth-child(1),
  td:nth-child(1) {